internal/cobraprompt/context.go

```diff
@@ -0,0 +1,24 @@
+package cobraprompt
+
+import (
+	"context"
+	"time"
+)
+
+// CobraMutableCtx is a workaround for https://github.com/spf13/cobra/issues/1109
+type CobraMutableCtx struct {
```

Comment:
It's not a good idea to implement the Context interface. Can't we use `context.Context` in the standard lib?

Reply:
We'll use it as is for now.